README.txt

```diff
@@ -57,17 +57,21 @@ Installation
 Any platform
 1. Download the latest zip release from https://github.com/openpreserve/fido/releases
 2. Unzip into some directory
-3. Open a command shell, cd to the directory that you placed the zip contents into and cd into folder 'fido'
-4. You should now be able to see the help text:
-   python fido.py -h
```

Comment:
This generates an error because python handles relative imports differently.
```
Traceback (most recent call last):
  File "fido.py", line 27, in <module>
    from . import __version__, CONFIG_DIR
ValueError: Attempted relative import in non-package
```
From the unzip directory, `python -m fido.fido` may work, but requires ole to be installed first.